From d4d1a5ece7ba37901f34a25b97f44119f5508954 Mon Sep 17 00:00:00 2001
From: Jiping Ma <jiping.ma2@windriver.com>
Date: Wed, 31 Aug 2022 21:56:56 -0700
Subject: [PATCH] mstflint: Rename libibmad-devel to libibmad-dev.

jenkins build failed with the error ": Depends: libibmad-devel
but it is not installable". As the error message indicates we
correct this error by renaming libibmad-devel to libibmad-dev.

Signed-off-by: Jiping Ma <jiping.ma2@windriver.com>
---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 0075a1d..32cb1db 100644
--- a/debian/control
+++ b/debian/control
@@ -2,7 +2,7 @@ Source: mstflint
 Section: net
 Priority: extra
 Maintainer: Gal Sivan <gsivan@unknown>
-Build-Depends: debhelper (>= 8.0.0), autotools-dev, libibmad-devel, zlib1g-dev, libssl-dev
+Build-Depends: debhelper (>= 8.0.0), autotools-dev, libibmad-dev, zlib1g-dev, libssl-dev
 Standards-Version: 3.9.2
 Homepage: http://www.mellanox.com
 #Vcs-Git: git://git.debian.org/collab-maint/mstflint.git
-- 
2.35.1

